### PR TITLE
Adds arguments to add shadows to showyboxes

### PR DESCRIPTION
**Pull request by jneug**
This update adds a new argument `shadow` to optionally add a solid colored shadow to a showybox.

### CODE REVIEW

File: showy.typ [hunk 1]
```diff
@@ -55,11 +55,40 @@
...
+        inset:  0pt,
+        move(
+          dx:-1 * shadow.offset.x,
+          dy:-1 * shadow.offset.y,
+          sbox
+        )
+      )
+    )
+  }
+  wrap(block(
```
Comment (Pablo-Gonzalez-Calderon) [inline showy.typ:91]: The `wrap()` function here is not closed. The parenthesis should be in the penultimate line of code in this file. I can not change it from here, so you should make a new commit changing it ;)

**Review by Pablo-Gonzalez-Calderon — COMMENTED** [review]
Looks good. I made a comment in `showy.typ` due to a missing parenthesis. After fixing that it'll be ready for merging. I'll take care of complementing the documentation ;)

**Comment by jneug** [comment]
Seems I accidentally deleted a line when fixing indentations. Should be fixed now.

I already added the new properties to the README and to the last example in line 52. If it works for you, you might want to retake the screenshot for that example. I didn't change version numbers or added to the changelog yet, though.

**Comment by Pablo-Gonzalez-Calderon** [comment]
Perfect! Thank you :)

**Pull request merged**